DBMS-Assignment-4/Question3.sql

```diff
@@ -0,0 +1,31 @@
+/*Identify the columns require indexing in order, product, category tables and create indexes.*/
+
+#table orders
+ALTER TABLE orders ADD INDEX index_bill_amount(bill_amount);
+
+ALTER TABLE orders ADD INDEX index_placed_date(placed_date);
+
+#table order_status
+ALTER TABLE order_status ADD INDEX index_status(status);
+
+ALTER TABLE order_status ADD INDEX index_order_id(order_id);
+
+ALTER TABLE order_status ADD INDEX index_product_id(product_id);

```

Comment:
Composite Primary keys are already indexed individually. No need to add indexing to them.